nemo_text_processing/text_normalization/en/taggers/roman.py

```diff
@@ -52,7 +68,7 @@ def __init__(self, deterministic: bool = True, lm: bool = False):
             + names
             + pynutil.insert("\"")
             + pynini.accep(" ")
-            + graph_teens @ default_graph
+            + regnal_graph 
```

Comment:
I noticed the `graph_teens` variable is no longer used here, I assume we meant to replace it with `regnal_graph`? i.e. basically making sure we didn't want both/all to be specified here? And checking if `graph_teens` is still used anywhere else?

I see the roman to spoken file (that was used in the graph_teens) has up to `MM ` specified, but they only used up to ordinal limit 19, I assume that is why we also have a short list?

Reply:
Yeah, the `graph_teens` var is used further down the file on line `111` so I didn't remove it. 

So the `roman_to_spoken` has up to `MM`, but `graph_teens` which I replaced only grabs from 2-19 for it's use cases. That's why I matched the short list use case in my implementation.